tools/traffic/generator.go

```diff
@@ -62,6 +63,8 @@ func (g *TrafficGenerator) StartTraffic(ctx context.Context) error {
 		return err
 	}
 
+	data = codec.ConvertByPaddingEmptyByte(data)
```

Comment:
Instead of putting this here, should we put this line in the client? 

https://github.com/bxue-l2/eigenda/blob/ca825a65d97f99fc92376bee78687c0431208782/clients/disperser_client.go#L85

and 

https://github.com/bxue-l2/eigenda/blob/ca825a65d97f99fc92376bee78687c0431208782/clients/disperser_client.go#L129-L130

Reply:
On second thought, hardcoding a specific implementation into the client would be bad. We should allow more flexible usage of the client. Please disregard this ^ comment.